lib/pbench/server/cache_manager.py

```diff
@@ -885,6 +885,8 @@ def create(self, tarfile: Path) -> Tarball:
         try:
             metadata = Tarball._get_metadata(tarfile)
             controller_name = metadata["run"]["controller"]
+        except MetadataError as exc:
+            raise MetadataError(tarfile, exc)
         except Exception as exc:
             raise MetadataError(tarfile, exc)
```

Comment:
I don't see any point to having two separate `except` clauses which perform exactly the same action.

We _could_ combine them together:
```
        except (Exception, MetadataError) as exc:
            raise MetadataError(tarfile, exc)
```
But that would be silly, given that any instance of `MetadataError` would match `Exception`, so there is no need to list it as well.

As long as I'm pointing out silly stuff, it looks to me like line 889 is raising another instance of the exception which it just caught.  All that that does is truncate the traceback and obscure the original source of the exception.  If we want to keep the `except` clause at line 888 (e.g., to keep it separate from the subsequent one), then its action should be a bare `raise`, to reraise the original `MetadataError` and keep the information intact.